website/source/docs/drivers/external/pot-task-driver.html.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: "docs"
+page_title: "Drivers: pot-task-driver"
+sidebar_current: "docs-drivers-community-pot-task-driver"
+description: |-
+  The Pot task driver is used to run pot (https://github.com/pizzamig/pot) containers using FreeBSD jails.
+---
+
+# Pot task Driver
+
+Name: `pot-task-driver`
+
+The Pot task driver provides an interface for using [pot][pot-github-repo] for dynamically running applications inside a FreeBSD Jail. 
+You can download the external pot-task-driver [here][pot-task-driver].
+
+## Task Configuration
+
+```hcl
+task "nginx-pot" {
+    driver = "pot"
+
+    config {
+      image = "https://pot-registry.zapto.org/registry/"
+      pot = "FBSD120-nginx"
+      tag = "1.0"
+      command = "nginx -g 'daemon off;'"
+      network_mode = "public-bridge" 
+      port_map = {
+        http = "80"
+      }
+      copy = [
+        "/root/index.html:/usr/local/www/nginx-dist/index.html",
+        "/root/nginx.conf:/usr/local/etc/nginx/nginx.conf"
+      ]
+      mount = [ 
+        "/tmp/test:/root/test",  
+      ]
+      mount_read_only = [
+        "/tmp/test2:/root/test2"
+      ]
+   }
+}
+```
+
+The pot task driver supports the following parameters: 
+
+* `image` - The url for the http registry from where to get the image.
+
+* `pot` - Name of the image in the registry.
+
+* `tag` - Version of the image.
+ 
+* `commad` - Command that is going to be executed once the jail is started.
+
+* `network_mode` - (Optional) Defines the network mode of the pot. Default: **"public-bridge"**
+
+  Possible values are:
+  
+  **"public-bridge"**  pot creates an internal virtual network with a NAT table where all traffic is going to be sent.
+  
+  **"host"** pot bounds the jail directly to a host port.
+
+* `port_map` - (Optional) Sets the port on which the application is listening inside of the jail. If not set, the application will inherit the port configuration from the image.
+
+* `copy` - (Optional) Copies a file from the host machine to the pot jail in the given directory.
+
+* `mount` - (Optional) Mounts a read/write folder from the host machine to the pot jail.
+
+* `mount_read_only` - (Optional) Mounts a read only directory inside the pot jail.
+
```

Comment:
I checked the task config in the source at [driver.go#L51-L66](https://github.com/trivago/nomad-pot-driver/blob/5b02965f3cfc06a28d631043efc4a76987c57ef9/driver/driver.go#L51-L66) and it looks like the `args`, `debug`, and `verbose` flags have been left off.